core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -70,7 +72,8 @@ class BaseUnit : INamed, IConstruction, CivilopediaText() {
         lines += "$strengthLine$movement${Fonts.movement}"
 
         if (replacementTextForUniques != "") lines += replacementTextForUniques
-        else for (unique in uniques)
+        else for (unique in uniques.filterNot { isConstructionInfoTable
+                && (Unique(it).placeholderText == "Requires []" || Unique(it).placeholderText == "Requires at least [] population") })
```

Comment:
????

Reply:
That means that the description  doesn't show construction require in constructioninfotable. If not, you will find the word like "consume 1 oil" will repeat in the table.

Reply:
I see
Then this should be 'else if(!isConstructionInfoTable) for(unique in uniqueObjects) ...